.travis.yml

```diff
@@ -61,5 +61,6 @@ script:
     # Setting up  Docker Registry is complete, let's do Buildah testing! 
     - make install.tools install.libseccomp.sudo all runc validate TAGS="apparmor seccomp containers_image_ostree_stub"
     - go test -c -tags "apparmor seccomp `./btrfs_tag.sh` `./libdm_tag.sh` `./ostree_tag.sh` `./selinux_tag.sh`" ./cmd/buildah
+    - go test -v -race $(go list ./... | grep -v vendor | grep -v tests | grep -v cmd)
```

Comment:
Can this be combined with line 63?

Reply:
I tried to merge them, but that caused the "./*.sh" files to fail before running any test.
I need to look more into it, but I guess this will be in another PR.

Reply:
Ah, I think I got it, this is going to run run_test.go below right?  If so, then it can't be combined with line 63.  @baude is setting up ginkgo gets for use under tests/e2e, we'll probably want to move run_test.go there eventually and convert it if necessary.

Reply:
On one hand  I can't move this to line 64 because multiple packages are not allowed when `-c` is set and on the other hand I need to set it with tags which leaves me confused :confused:

Reply:
I decided not to add anything to `.travis.yml` as I put it under a lot of load.
I wish there was a way to reproduce `Travis-CI` on my local machine. :thinking: